### PR TITLE
Added instances related to Administration Type

### PR DESCRIPTION
**Pull request by MaaikevS**
Related to issue #135  and PR #137 

The definitions might need some tweaking to make it more generalised. I was going back and forth on whether the definition needs to be related to 'administration' or any type of procedure. Feel free to provide some suggestions here :)

### CODE REVIEW

File: instances/administrationType/intraperitoneal.jsonld [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/administrationType/intraperitoneal",
```
Comment (lzehl) [inline instances/administrationType/intraperitoneal.jsonld]: I would be more specific and say `intraperitonealInjection` and change respectively the definition and name (which is anyway not correct; copy / past error I assume)

File: instances/administrationType/intravenous.jsonld [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/administrationType/intravenous",
```
Comment (lzehl) [inline instances/administrationType/intravenous.jsonld]: also here I would be more specific: `intravenousInjection` OR `intravenousInsertion` OR `intravenousAdministration`

File: instances/administrationType/intraperitoneal.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/administrationType/intraperitoneal.jsonld]: definition suggestion: "An 'intraperitoneal injection' is the administration of a substance into the peritoneum (body cavity) via a needle or tube."

File: instances/administrationType/oral.jsonld [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/administrationType/oral",
```
Comment (lzehl) [inline instances/administrationType/oral.jsonld]: oralAdministration

File: instances/administrationType/oral.jsonld [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/AdministrationType",
+  "definition": "'Oral' administration is a route of administration where a substance is taken via the mouth.",
```
Comment (lzehl) [inline instances/administrationType/oral.jsonld]: "In an 'oral administration' a substance is taken through the mouth."

File: instances/administrationType/subcutaneous.jsonld [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/administrationType/subcutaneous",
```
Comment (lzehl) [inline instances/administrationType/subcutaneous.jsonld]: subcutaneousInjection
(cf. all comments/suggestions in intraperitoneal)

File: instances/administrationType/intravenous.jsonld (hunk 2 above)
Comment (lzehl) [reply instances/administrationType/intravenous.jsonld]: (cf. all comments/suggestions for intraperitoneal)

File: instances/administrationType/oral.jsonld [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "intravenous",
```
Comment (lzehl) [inline instances/administrationType/oral.jsonld]: wrong name

File: instances/administrationType/intravenous.jsonld [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/AdministrationType",
+  "definition": "An 'intravenous' procedure takes place within a vein or veins.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "intravenous",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "i.v."
```
Comment (lzehl) [inline instances/administrationType/intravenous.jsonld]: also IV

File: instances/administrationType/intraperitoneal.jsonld [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/AdministrationType",
+  "definition": "An 'intraperitoneal' procedure or administration takes place within the peritoneal cavity (the area that contains the abdominal organs).",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "intravenous",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "i.p."
```
Comment (lzehl) [inline instances/administrationType/intraperitoneal.jsonld]: also IP

File: instances/administrationType/oral.jsonld [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+  "preferredOntologyIdentifier": null,
+  "synonym": null
```
Comment (lzehl) [inline instances/administrationType/oral.jsonld]: per os
PO
p.o.

File: instances/administrationType/subcutaneous.jsonld [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/AdministrationType",
+  "definition": "A 'subcutaneous' procedure or administration takes place beneath, or under, all the layers of the skin.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "intravenous",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "s.c."
```
Comment (lzehl) [inline instances/administrationType/subcutaneous.jsonld]: also SC
SC injection

**Review by lzehl — CHANGES_REQUESTED** [review]
@MaaikevS sorry I did not see this before, but actually I would register all of those as techniques with the suggested name changes.

@UlrikeS91 would you agree?

**Comment by MaaikevS** [comment]
@lzehl I think your suggestion makes sense, so I moved them and updated with the requested changes.

**Review by lzehl — APPROVED** [review]

**Pull request merged**